common/spec/dependabot/clients/azure_spec.rb

```diff
@@ -181,6 +181,107 @@
     end
   end
 
+  describe "#create_pull_request" do
+    subject do
+      client.create_pull_request(
+        pr_name, source_branch, target_branch,
+        pr_description, labels, reviewers, work_item
+      )
+    end
+
+    let(:pr_name) { "test-create-pr" }
+    let(:source_branch) { "feature/test-create-pr" }
+    let(:target_branch) { "main" }
+    let(:pr_description) { "PR description" }
+    let(:labels) { [] }
+    let(:reviewers) { [] }
+    let(:work_item) { nil }
+    let(:create_pull_request_url) do
+      base_url + "/_apis/git/repositories/#{source.unscoped_repo}/pullrequests?api-version=5.0"
+    end
+
+    context "when response is 200" do
+      response_body = fixture("azure", "create_pull_request_details.json")
+
+      before do
+        @request_body = "{\"sourceRefName\":\"refs/heads/#{source_branch}\","
+        + "\"targetRefName\":\"refs/heads/#{target_branch}\",\"title\":\"#{pr_name}\","
+        + "\"description\":\"#{pr_description}\",\"labels\":[],\"workItemRefs\":[],\"reviewers\":[]}"
+
+        stub_request(:post, create_pull_request_url).
+          with(basic_auth: [username, password], body: @request_body).
+          to_return(status: 200, body: response_body)
+      end
+
+      specify { expect { subject }.to_not raise_error }
```

Comment:
We already implicitly test this doesn't raise an error in the next test, so I think it's safe to drop this 👍